sklearn/datasets/tests/test_lfw.py

```diff
@@ -147,16 +160,27 @@ def test_load_fake_lfw_people():
 
 @raises(ValueError)
 def test_load_fake_lfw_people_too_restrictive():
-    load_lfw_people(data_home=SCIKIT_LEARN_DATA, min_faces_per_person=100)
+    with warnings.catch_warnings():
+        load_lfw_people(data_home=SCIKIT_LEARN_DATA, min_faces_per_person=100)
 
 
 @raises(IOError)
 def test_load_empty_lfw_pairs():
-    load_lfw_pairs(data_home=SCIKIT_LEARN_EMPTY_DATA)
+    with warnings.catch_warnings():
+        load_lfw_pairs(data_home=SCIKIT_LEARN_EMPTY_DATA)
+
+
+def test_load_lfw_pairs_deprecation():
+    msg = ("Function 'load_lfw_pairs' has been deprecated in 0.17 and will be "
+           "removed in 0.19."
+           "Use fetch_lfw_pairs(download_if_missing=False) instead.")
+    assert_warns_message(DeprecationWarning, msg, load_lfw_pairs,
+                         data_home=SCIKIT_LEARN_DATA)
 
 
 def test_load_fake_lfw_pairs():
-    lfw_pairs_train = load_lfw_pairs(data_home=SCIKIT_LEARN_DATA)
+    with warnings.catch_warnings():
+        lfw_pairs_train = load_lfw_pairs(data_home=SCIKIT_LEARN_DATA)
```

Comment:
Ah. Sorry then my previous guidance was wrong. I thought there were tests for both. Please rename all tests to use ``fetch_lfw_pairs``. If you like, you can do a for loop in the tests over both functions, but I don't think this is necessary.